deployment/ansible/development/group_vars/all.sample.yml

```diff
@@ -49,6 +51,24 @@ interpreters:
     # Point to the ipaddress/domain name of InaSAFE Realtime processor
     ipaddress: *inasafe_django_host_ip
 
+# Facts needed for InaSAFE Realtime
+# This is to setup Realtime Hazard services
+inasafe_realtime:
```

Comment:
This section to customize inasafe_realtime service.
@ismailsunni will have to create or adapt similar one for headless.

Reply:
Noted.